styleguide/source/_patterns/03-organisms/masthead/masthead~with-subtitle-as-centered.json

```diff
@@ -0,0 +1,45 @@
+{
+  "masthead": {
+    "centered": true,
+    "link": {
+      "title": "Category link",
+      "href": "#",
+      "text": "Delivering care",
+      "target": "_self",
+      "class": "ama__masthead__content__link"
+    },
+    "heading": {
+      "text": "JAMA Network to launch fully open-access journal"
+    },
+    "subtitle": {
+      "text": "Example text for a subtitle, lorem ipsum sit dolor"
```

Comment:
Let's match the text here to the Zeplin

Text describing what this rule means and how the AMA has provided the below to help. Lorem Ipsum. Alsip Clark Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam et felis vel massa consequat congue. Praesent viverra feugiat arcu aliquam posuere. Integer non luctus urna, quis tincidunt nisi. Proin ullamcorper velit in vulputate sodales. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Cras tincidunt id nisi ac ornare. Nam vel fermentum ligula.

Reply:
Added Zeplin text and fixed the font size and weight of the subtitle.